### DISCUSSION TITLE
Add DeliveryDate to Order search

### DISCUSSION THREAD

**Discussion by eriksallberg** [FortnoxAB/csharp-api-sdk#300] [Ideas]
Would it be possible to add a FromDeliveryDate and ToDeliveryDate to the Order search so that you can search for orders in the same way you can with the order date using the FromDate and ToDate properties?

For example, I need to retrieve all orders with a delivery date from within the last week. An order can be placed up to six months before its delivery date, so I have to retrieve all orders from the last six months and then filter out the orders that have a delivery date from last week. In total, that's 57 requests (5700 orders in the last six months, 100 orders per request) that could be just one request if you could search on the delivery date directly.

**Comment by bpaolo71** [comment]
According to the documentation it is not possible to filter by delivery date: https://apps.fortnox.se/apidocs#tag/Orders.
